libraries/stdlib/genglsl/mx_smoothstep_float.glsl

```diff
@@ -1,9 +1,9 @@
 void mx_smoothstep_float(float val, float low, float high, out float result)
 {
-    if (val <= low)
-        result = 0.0;
-    else if (val >= high)
+    if (val >= high)
         result = 1.0;
+    else if (val <= low >)
```

Comment:
There's a typo at the end of the line ;)

Reply:
good catch - but also - how did the tests even pass?!? is this even legal code?